server/datastore/mysql/software.go

```diff
@@ -2826,6 +2826,36 @@ func hostVPPInstalls(ds *Datastore, ctx context.Context, hostID uint, globalOrTe
 	return vppInstalls, nil
 }
 
+func hostInstalledVpps(ds *Datastore, ctx context.Context, hostID uint) ([]*hostSoftware, error) {
+	vppInstalledStmt := `
```

Comment:
We're mising label scoping here :(

If the VPP app got installed and later got dropped out of label scope, it should show up as in inventory and neither as installed nor as available for install.

This will continue being an issue when we revise host install scoping, though at that point self-service will be filtered by labels rather than overall availability for install.

Reply:
Should I cover the scoping as part of https://github.com/fleetdm/fleet/issues/28328 in a new PR?

Or would you prefer if I do it in this PR for vpp installs and then cover the rest of the cases missed in the new PR?

Reply:
Can handle scoping in a new PR. I'll give this one a thumbs-up momentarily for merge with the knowledge that we have a separate bug tracked for that work.

Reply:
https://github.com/fleetdm/fleet/pull/28432